tensorflow/core/kernels/slice_op_cpu_impl_6.cc

```diff
@@ -1,18 +0,0 @@
-/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
The static part can be compiled into multiple files. Each branch of the switch statement goes into a separate file.